db/orders.js

```diff
@@ -0,0 +1,66 @@
+const client = require("./client");
+
+async function getOrderById(id) {
+  try {
+    const {
+      rows: [order],
+    } = await client.query(`
+            SELECT *
+            FROM orders
+            WHERE id=id;
+        `);
+    return order;
+  } catch (error) {
+    throw error;
```

Comment:
use console.log instead of throwing inside of the catch blocks (or find some other way to handle errors)